<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="convertFormat">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Convert video and audio formats</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="changeResolution">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Change video resolution</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addRemoveAudio">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Add audio to video or remove audio from it</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectVideos">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect several video files to each other</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reduceVolume">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Reduce video volume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletePartVideo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete parts of the video</string>
    </property>
   </widget>
   <zorder>changeResolution</zorder>
   <zorder>addRemoveAudio</zorder>
   <zorder>connectVideos</zorder>
   <zorder>play</zorder>
   <zorder>reduceVolume</zorder>
   <zorder>deletePartVideo</zorder>
   <zorder>convertFormat</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
